### DISCUSSION TITLE
U5パワーアンプはSOP-8パッケージのNJM2113Mも使えるようにしませんか？

### DISCUSSION THREAD

**Discussion by y-ohe** [algyan/XIAOGYAN#19] [Ideas]
U5パワーアンプはNJM2113DのDIP8パッケージ品になっていますが、SOP-8パッケージのNJM2113M(I-10981)も使えるようにしませんか？

**Comment by matsujirushi** [comment]
アイデア、ありがとうございます！

A品だったので無視していましたが、、、フットプリント追加するだけなので、やってみます！

**Reply by matsujirushi** [reply]
チップはギリ入りました。
![image](https://user-images.githubusercontent.com/8079404/211125391-089e72f8-2c4b-4069-83de-f25ef6e2e5dc.png)

**Comment by matsujirushi** [comment]
配線完了。
![image](https://user-images.githubusercontent.com/8079404/211126428-91969a13-a1b0-45ec-b81a-2e8bb722d1dd.png)
